anta/tests/routing/bgp.py

```diff
@@ -11,7 +11,8 @@
 from ipaddress import IPv4Address, IPv4Network, IPv6Address
 from typing import Any, List, Optional, Union, cast
 
-from pydantic import BaseModel, Field, PositiveInt, model_validator, utils
+from pydantic import BaseModel, Field, PositiveInt, model_validator
+from pydantic.v1.utils import deep_update
```

Comment:
can we avoid this?

Reply:
Pydantic sends a warning asking to change the import: https://docs.pydantic.dev/latest/migration/

Reply:
yeah we may need to keep an eye on this one..